bouncer/tests/btc_vault_swap.ts

```diff
@@ -112,8 +114,25 @@ async function testVaultSwap(depositAmountBtc: number, brokerUri: string, destin
   testBtcVaultSwap.log(`Balance increased, Vault Swap Complete`);
 }
 
+async function openPrivateBtcChannel(brokerUri: string) {
+  // TODO: Use chainflip SDK instead so we can support any broker uri
+  assert.strictEqual(brokerUri, '//BROKER_1', 'Support for other brokers is not implemented');
+
+  // TODO: use chainflip SDK to check if the channel is already open
+  try {
+    await jsonRpc('broker_open_private_btc_channel', [], 'http://127.0.0.1:10997');
```

Comment:
Does the jsonRpc util not have a constant (or similar) to avoid having to explicitly pass the api url/port??

Reply:
Didnt seem worth adding when the code will be replaced later by the chainflip SDK.